.ncurc.json

```diff
@@ -0,0 +1,3 @@
+{
+	"reject": ["@mysten/sui.js"]
```

Comment:
Disable auto-upgrade of this module. We'll update it manually to make it match the version the latest shinami release depends on.